### PR TITLE
[FIX] rule norm

### PR DESCRIPTION
**Pull request by lg-epitech**
actually coding style funny enough

### CODE REVIEW

File: gertrude_base/free_madness.c
Comment (AshDev05) [inline gertrude_base/free_madness.c:1]: why not keep the if statements?

File: gertrude_base/gertrude.c [hunk 1]
```diff
@@ -65,10 +65,12 @@ int main(int ac, char **av)
...
     ger.epitech = 0;
     ger.variables = NULL;
-    ger.rule_names = NULL;
+    ger.rule_names = malloc(sizeof(NULL));
+    ger.rule_names[0] = NULL;
     ger.deps = NULL;
     ger.cmds = NULL;
-    ger.rules = NULL;
+    ger.rules = malloc(1);
+    strcpy(ger.rules, "\0");
```
Comment (AshDev05) [inline gertrude_base/gertrude.c:73]: why not strdup here?

**Review by AshDev05 — COMMENTED** [review]
There are some inconsistencies please reply to the comments coz brain not braining rn

File: gertrude_base/free_madness.c
Comment (lg-epitech) [reply gertrude_base/free_madness.c:1]: because free already handles if you give it NULL it just returns right away

Comment (lg-epitech) [reply gertrude_base/free_madness.c:1]: idk when I saw it my adhd said no, if you want to keep them go ahead

File: gertrude_base/gertrude.c (hunk 1 above)
Comment (lg-epitech) [reply gertrude_base/gertrude.c:73]: I wanted to do that but I forgor, i already did it at some other point in the new rules code. I suggest you add that in the pr.

File: gertrude_base/free_madness.c
Comment (AshDev05) [reply gertrude_base/free_madness.c:1]: No it's fine thanks for the explanation!

File: gertrude_base/gertrude.c (hunk 1 above)
Comment (AshDev05) [reply gertrude_base/gertrude.c:73]: 👍

**Review by AshDev05 — APPROVED** [review]
All good and consistent I like it

**Pull request merged**